src/averageAges.js

```diff
@@ -20,6 +20,12 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredMales = (!century)
+    ? peopleFilter(people, person => person.sex === 'm')
+    : peopleFilter(people, person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century);
```

Comment:
As for me moving filter method to separated function is not so good idea, but ok